src/compiler/diagnosticMessages.json

```diff
@@ -888,6 +888,10 @@
         "category": "Error",
         "code": 2322
     },
+    "Type '{0}' is not comparable with type '{1}'.": {

```

Comment:
I prefer 'comparable to' here. A little research shows that grammarians think they're about equally common, while Google's hideously unreliable hit count shows 4 times more hits for 'comparable to'*.

But the function is also named `isTypeComparableTo`, so I think there's precedent. :)

*Google hit counts are the laziest way to do quantitative linguistics.